client/analytics/components/report-summary/test/index.js

```diff
@@ -49,86 +50,94 @@ describe( 'ReportSummary', () => {
 		);
 	}
 
-	test( 'should set the correct prop values for the SummaryNumber components', () => {
-		const reportChart = renderChart( 'number', 1000.5, 500.25 );
-		const summaryNumber = reportChart.find( 'SummaryNumber' );
+	test( 'should set the correct prop values for the SummaryNumber components', async () => {
+		renderChart( 'number', 1000.5, 500.25 );
 
-		expect( summaryNumber.props().value ).toBe( '1,000.5' );
-		expect( summaryNumber.props().prevValue ).toBe( '500.25' );
-		expect( summaryNumber.props().delta ).toBe( 100 );
-	} );
+		expect( screen.getByText( '1,000.5' ) ).not.toBeEmptyDOMElement();
```

Comment:
I think replacing `.not.toBeEmptyDOMElement()` with `.toBeInTheDocument()` will make this test more clear. `getBy*` was chosen because the expectation is that the text can be found - the test should fail if it cannot be.